src/App.tsx

```diff
@@ -1,10 +1,14 @@
+import { BrowserRouter } from 'react-router-dom';
+import { Navigation } from './components/Navigation/Navigation';
 import './App.css';
```

Comment:
I am not a big fan of declaring only global css because we will have in future name collisions.
We can use modular css(with some hash in front of the class name), or we will introduce some CSS-in-JS lib like `styled-components` or we can use preprocess css like SASS(SCSS) with mixins(or BEM structure).
You can choose whatever you like, I am fine with all :)

Reply:
I will use `styled-components`. I agree with you on this, it will be refactored according to `styled-components`.